common/script_values/babylon_values.txt

```diff
@@ -748,3 +748,9 @@ babylon_player_construction_bonus_multiplier = {
 		}
 	}
 }
+
+babylon_greener_grass_modifier_multiplier = {
+	add = 1
+	value = free_arable_land 
```

Comment:
proper would be first define the value then add 1

Reply:
Was this resolved by @Danarca ?